dev/unix/volta-install.sh

```diff
@@ -129,6 +129,8 @@ parse_os_info() {
     Linux)
       if [ "$arch" == "x86_64" ]; then
         echo "linux"
+      elif [ "$arch" == "aarch64" ]; then
+        echo "linux-aarch64"
       else
         error "Releases for non x64 architectures are not currently supported."
```

Comment:
Thank you for this. Might be worth updating the message as well

Reply:
Hi @Fydon would you point me which message I need to update?

Reply:
Sure. For macOS there is no error text, so perhaps the same is true for Linux when ARM64 is supported. Otherwise the error message should be something like "Only x64 and ARM64 are currently supported". I believe that x64 is x86_64 and not all architectures ending in 64.